src/platform/datapath_linux.c

```diff
@@ -2179,6 +2179,11 @@ QuicDataPathWorkerThread(
     QUIC_DATAPATH_PROC_CONTEXT* ProcContext = (QUIC_DATAPATH_PROC_CONTEXT*)Context;
     QUIC_DBG_ASSERT(ProcContext != NULL && ProcContext->Datapath != NULL);
 
+    QuicTraceLogInfo(
+        DatapathWorkerThread,
```

Comment:
This is the event name, and needs to be unique, per string (`"[ udp][%p] Worker start"`) used. Go ahead and name this `DatapathWorkerThreadStart` and the other one `DatapathWorkerThreadStop`.